example/.kube.yml

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
It looks like the secrets not used in the deployment.

Reply:
Good call.

Reply:
Fixed*